app/models/manageiq/providers/amazon/inventory/parser/storage_manager/ebs.rb

```diff
@@ -76,14 +76,11 @@ def link_volume_to_disk(volume_hash, attachments)
 
       dev = File.basename(a['device'])
 
-      disk_hash = {
```

Comment:
so, the changes here in `storage_manager/ebs.rb ` and `inventory_collection_default/cloud_manager.rb` are required to make the changes in `collector/target_collection.rb` to work?